Packs/rasterize/Integrations/rasterize/README.md

```diff
@@ -30,6 +30,17 @@ If you want to set the language to en-US, use en-GB instead.
 * Rasterize Mode: It is possible to rasterize either via Chrome WebDriver or Chrome Headless CLI. WebDriver supports more options than Headless CLI. Such as support for the `offline` option in the `rasterize-emails` command. There are some urls that do not rasterize well with WebDriver and may succeed with Headless CLI. Thus, it is recommended to use the `WebDriver - Preferred` mode, which will use WebDriver as a start and fallback to Headless CLI if it fails.
```

Comment:
Rasterize Mode: You can rasterize through the Chrome WebDriver or the Chrome Headless CLI. WebDriver supports more options than Headless CLI, such as the `offline` option in the `rasterize-emails` command. Some URLs do not rasterize well with WebDriver but may succeed with Headless CLI. We recommend using the `WebDriver - Preferred` mode, which uses WebDriver to start, but with a fallback to Headless CLI if it fails.